File: app/src/main/java/com/example/picobusinesstouser/TobHookModule.java
package com.example.picobusinesstouser;

import android.util.Log;

import java.util.Objects;

import de.robv.android.xposed.IXposedHookLoadPackage;
import de.robv.android.xposed.XC_MethodHook;
import de.robv.android.xposed.XposedHelpers;
import de.robv.android.xposed.callbacks.XC_LoadPackage;

public class TobHookModule implements IXposedHookLoadPackage {
    @Override
    public void handleLoadPackage(XC_LoadPackage.LoadPackageParam lpparam) {
        if (!Objects.equals(lpparam.packageName, "com.pvr.shortcut")) return;

        try {
            var dockUtilsClass = XposedHelpers.findClass(
                    "com.pvr.shortcut.utils.DockUtils",
                    lpparam.classLoader
            );

            var hook = new XC_MethodHook() {
                @Override
                protected void beforeHookedMethod(MethodHookParam param) throws Throwable {
                    super.beforeHookedMethod(param);
                }
            };

            XposedHelpers.findAndHookMethod(
                    dockUtilsClass,
                    "isSupportToB",
                    hook
            );
        } catch (Exception e) {
            Log.e("TobHookModule", String.valueOf(e.getMessage()));
        }
    }
}

File: app/src/main/java/com/example/picobusinesstouser/TobHookModule.kt
package com.example.picobusinesstouser
import android.util.Log
import de.robv.android.xposed.IXposedHookLoadPackage
import de.robv.android.xposed.XC_MethodHook
import de.robv.android.xposed.XposedHelpers
import de.robv.android.xposed.callbacks.XC_LoadPackage

class TobHookModule : IXposedHookLoadPackage {
    override fun handleLoadPackage(lpparam: XC_LoadPackage.LoadPackageParam) {
        if (lpparam.packageName != "com.pvr.shortcut") return

        try {
            val dockUtilsClass = XposedHelpers.findClass(
                "com.pvr.shortcut.utils.DockUtils",
                lpparam.classLoader
            )

            XposedHelpers.findAndHookMethod(
                dockUtilsClass,
                "isSupportToB",
                object : XC_MethodHook() {
                    override fun beforeHookedMethod(param: MethodHookParam) {
                        param.result = false
                    }
                }
            )
        } catch (e: Exception) {
            Log.e("TobHookModule", "${e.message}")
        }
    }
}
